spec/models/scheduler_spec.rb

```diff
@@ -20,4 +20,32 @@
     expect(Date.parse(plant_2.waterings.third.water_time.to_s)).to eq(Date.parse(5.days.from_now.localtime.to_s))
     expect(Date.parse(plant_2.waterings.fourth.water_time.to_s)).to eq(Date.parse(7.days.from_now.localtime.to_s))
   end
+
+  it '.generate_waterings_for_a_week_from_today' do
+    def a_day_goes_by(plants)
+      plants.each do |plant|
+        plant.waterings.each do |watering|
+          #similating the passage of time
```

Comment:
Not sure if we want to leave this commented out code in there, but if we do, I think it's "simulating" 🤓